share/completions/ng.fish

```diff
@@ -0,0 +1,21 @@
+complete -c ng -f -d "The Angular CLI"
+
+complete -c ng -n __fish_use_subcommand -a "add" -d "Adds support for an external library to your project."
+complete -c ng -n __fish_use_subcommand -a "analytics" -d "Configures the gathering of Angular CLI usage metrics. See https://angular.io/cli/usage-analytics-gathering."
+complete -c ng -n __fish_use_subcommand -a "build" -d "Compiles an Angular app into an output directory named dist/ at the given output path. Must be executed from within a workspace directory."
```

Comment:
Just

"Compile to dist/"?

>Must be executed from within a workspace directory

That looks like a condition you could check. Or, if it's not feasible, just leave it and the user'll see the error and correct.